decidim-core/app/views/decidim/widgets/show.js.erb

```diff
@@ -0,0 +1,3 @@
+document.write('<link type="text/css" rel="stylesheet" media="all" href="<%= asset_url 'decidim/widget.css' %>"></link>');
```

Comment:
I wouldn't implement it this way but rather with an autogrowing iframe because of the following reasons:

* In the current implementation, the widget's CSS and JS are sensible to the hosts' and will break each other.
* iFrames can load their own resources without running into CORS issues (typography, extra JS, etc)
* iFrames are completely isolated and you could safely use cookies (if you wanted) enabling new functionality.
* Relative URLs when using `document.write` won't work.

By using `iframes`, the widgets don't have any extra knowledge more than a special layout and session constraints.

The only drawback I know is that an iframe's background won't blend with the website's (so the embedder will have to find a solution) and doesn't autogrow. For this last issue, there's easy ways to handle it:

http://stackoverflow.com/a/9976309